src/algebra/order/ring.lean

```diff
@@ -1021,6 +1021,35 @@ instance linear_ordered_ring.to_linear_ordered_add_comm_group [s : linear_ordere
   linear_ordered_add_comm_group α :=
 { .. s }
 
+section linear_ordered_semiring
+variables [linear_ordered_semiring α] {a b c : α}
+
+lemma le_of_mul_le_of_one_le {a b c : α} (h : a * c ≤ b) (hb : 0 ≤ b) (hc : 1 ≤ c) : a ≤ b :=
+by haveI := @linear_order.decidable_le α _; exact
```

Comment:
oh, this issue pops up here! it's definitely worth fixing this somehow, I think `local attribute [instance] linear_ordered_semiring.decidable_le` is probably worth having, if not the global one

Reply:
Done!